client/battle/BattleWindow.cpp

```diff
@@ -733,6 +749,32 @@ std::optional<uint32_t> BattleWindow::getQueueHoveredUnitId()
 	return queue->getHoveredUnitIdIfAny();
 }
 
+void BattleWindow::endWithAutocombat() 
```

Comment:
Perhaps add confirmation prompt? I think HD mod does have it and generally makes sense since it might have rather serious effects.

Reply:
Good idea! Done.